examples/sudoku/ts/src/sudoku-zkapp.ts

```diff
@@ -93,17 +104,23 @@ class SudokuZkApp extends SmartContract {
   }
 }
 
-// setup
-const Local = Mina.LocalBlockchain();
-Mina.setActiveInstance(Local);
-const account1 = Local.testAccounts[0].privateKey;
+// helpers
+function createLocalBlockchain(): PrivateKey {
+  let Local = Mina.LocalBlockchain();
+  Mina.setActiveInstance(Local);
 
-const zkAppPrivateKey = PrivateKey.random();
-let zkAppAddress = zkAppPrivateKey.toPublicKey();
+  const account = Local.testAccounts[0].privateKey;
+  return account;
+}
 
-async function deploy(sudoku: number[][]) {
-  let tx = await Mina.transaction(account1, () => {
-    Party.fundNewAccount(account1);
+async function deploy(
+  sudoku: number[][],
+  account: PrivateKey,
+  zkAppAddress: PublicKey,
+  zkAppPrivateKey: PrivateKey
+) {
+  let tx = await Mina.transaction(account, () => {
```

Comment:
this transaction block shouldn't duplicate initializing the state. if we go with the extra initialize method I proposed above, then let's just call that method here instead of doing `zkApp.sudokuHash.set(sudokuInstance.hash()); zkApp.isSolved.set(Bool(false));`